### PR TITLE
Add settings feature toggle in org settings

### PR DESCRIPTION
**Pull request by malted**
image

### CODE REVIEW

**Review by sarthaktexas — CHANGES_REQUESTED** [review]
Add a background

**Comment by maxwofford** [comment]
I'm fine with this merging without a background just for now because it's a feature flag! We should clarify though, can anyone toggle the feature flag currently?

**Comment by maxwofford** [comment]
2 drafts you can use:

amaziograph_DBC9F70E-9C41-45A4-A444-0EB6BE1EE33B amaziograph


amaziograph_DBC9F70E-9C41-45A4-A444-0EB6BE1EE33B amaziograph

**Comment by maxwofford** [comment]
These aren't relevant to the feature, but still fun patterns:

amaziograph_5FB89CDD-0783-4282-9984-CBF06D26DEBD amaziograph
amaziograph_45819717-5BC1-4088-B6F9-54AD1466CDC7 amaziograph

**Comment by malted** [comment]
image
image

File: app/views/events/edit.html.erb
Comment (garyhtou) [inline app/views/events/edit.html.erb]: Can we have a more descriptive copy here? Maybe even link to the `/roles` page

```erb
<%= link_to "learn more about the roles", roles_path %>
```

**Review by garyhtou — COMMENTED** [review]
What happens to member roles when the feature gets disabled? Should they become managers? or silently stay as members? (mentioned in https://github.com/hackclub/hcb/issues/5533)

**Comment by malted** [comment]
image

image


All users and pending invites automatically become managers after the user accepts this dialogue

**Comment by garyhtou** [comment]
@sarthaktexas can you do another review on this.

good to merge after @sarthaktexas's approval

File: app/views/events/edit.html.erb
Comment (sarthaktexas) [inline app/views/events/edit.html.erb]:
```suggestion
    Disallow certain users from performing critical financial actions!
```

File: app/assets/stylesheets/components/_features.scss
Comment (sarthaktexas) [inline app/assets/stylesheets/components/_features.scss]:
```suggestion
      opacity: 0.3;
```

**Review by sarthaktexas — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/components/_features.scss
Comment (sarthaktexas) [reply app/assets/stylesheets/components/_features.scss]: Your choice if you want to keep this or not:
Screenshot 2024-03-18 at 2 15 16 PM

Comment (malted) [reply app/assets/stylesheets/components/_features.scss]: Looks good; cheers!

**Comment by malted** [comment]
I work at a bank and spelled financial wrong 🎀

**Review by sarthaktexas — APPROVED** [review]

**Pull request merged**
